src/META-INF/plugin.xml

```diff
@@ -48,11 +48,11 @@
             </tr>
             <tr>
               <td align="right"><b>Support:</b></td>
-              <td align="left"><a style="color:white;" href="http://community.haxe.org/">Community Discourse</a></td>
+              <td align="left"><a style="color:white;" href="https://community.haxe.org/">Community Discourse</a></td>
             </tr>
             <tr>
               <td align="right"><b>Enterprise Support:</b></td>
-              <td align="left"><a style="color:white;" href="http://bishtonsoftwaresolutions.com/">Bishton Software Solutions</a></td>
+              <td align="left"><a style="color:white;" href="https://bishtonsoftwaresolutions.com/">Bishton Software Solutions</a></td>
             </tr>
```

Comment:
We can drop this stanza as there are no customers for Enterprise Support at this time, and I'm not actively involved any more.